ur_robot_driver/src/dashboard_client_ros.cpp

```diff
@@ -74,6 +74,17 @@ DashboardClientROS::DashboardClientROS(const rclcpp::Node::SharedPtr& node, cons
 
   // Stop program execution on the robot
   stop_service_ = createDashboardTriggerSrv("stop", "stop\n", "Stopped");
+
+  // Dismiss a protective stop to continue robot movements. NOTE: It is the responsibility of the user to ensure the
+  // cause of the protective stop is resolved before calling this service.
+  unlock_protective_stop_service_ =
+      createDashboardTriggerSrv("unlock_protective_stop", "unlock protective stop\n", "Protective stop releasing");
+
+  using namespace std::placeholders;
```

Comment:
I'd prefer not to have this here. It's just not something I've seen very often. Go ahead and spell it out completely:

std::placeholders::_1

Reply:
@tylerjw do you agree with me?  (Tyler is our C++ guru)

Reply:
This is sort of a special case.  The placeholders are annoying and this is nowhere near as bad as `using namespace std;`.

Reply:
@AndyZe  I will remove it for now, and leave the full spelling e.g. `std::placeholders::_1, std::placeholders::_2 `